### DISCUSSION TITLE
Bugs and errors?

### DISCUSSION THREAD

**Discussion by tedmarov** [remoteintech/remote-jobs#1803] [General]
I have found lots of links that are broken, not working, or otherwise closed. I can do some work and fix it up with PRs, if that's okay. If not, no worries.

**Comment by Ki-er** [comment]
We welcome housekeeping PRs!

**Comment by AndersonVes** [comment]
The voxy link is broken. This is working: https://jobs.lever.co/voxy

**Reply by LilianaTS** [reply]
<img width="1612" height="419" alt="image" src="https://github.com/user-attachments/assets/502f28ac-8ef7-47bb-af37-9a42c53173d2" />
That isn't working for me

**Comment by LilianaTS** [comment]
These no longer exist: 
https://remoteintech.company/companies/no-code-no-problem/
https://remoteintech.company/companies/18f/

I can't find a way to create an issue that's for Deletion of companies

**Comment by dougaitken** [comment]
hey all, i've been working on recurring job to check links and such for link rot. 18f was removed, I'll check [no-code-no-problem/](https://remoteintech.company/companies/no-code-no-problem/). 

Anything else, PRs are welcome to fix things (including updating the `updatedAt` date in the profile), removing dead links, etc. Or just open an Issue using Bug report template. I might create a new template for "fixing existing profile".
